handsontable/tslint.json

```diff
@@ -0,0 +1,7 @@
+{ 
+  "extends": "../tslint.json",
+  "rules": {
+    "forbidden-types": false,
+    "no-single-declare-module": false
```

Comment:
Why was it necessary to disable these rules? You could convert the forbidden types and remove the ```declare module```